CRITICAL - 2020-11-23 21:20:47 --> Unable to connect to the database.
#0 C:\xampp\htdocs\Codeigniter\system\Database\BaseConnection.php(646): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\xampp\htdocs\Codeigniter\system\Database\BaseBuilder.php(1860): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\xampp\htdocs\Codeigniter\system\Model.php(1818): CodeIgniter\Database\BaseBuilder->get()
#3 C:\xampp\htdocs\Codeigniter\app\Controllers\BaseController.php(49): CodeIgniter\Model->__call('get', Array)
#4 C:\xampp\htdocs\Codeigniter\system\CodeIgniter.php(891): App\Controllers\BaseController->initController(Object(CodeIgniter\HTTP\IncomingRequest), Object(CodeIgniter\HTTP\Response), Object(CodeIgniter\Log\Logger))
#5 C:\xampp\htdocs\Codeigniter\system\CodeIgniter.php(399): CodeIgniter\CodeIgniter->createController()
#6 C:\xampp\htdocs\Codeigniter\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\Codeigniter\index.php(46): CodeIgniter\CodeIgniter->run()
#8 {main}
CRITICAL - 2020-11-23 21:31:00 --> syntax error, unexpected ')'
#0 C:\xampp\htdocs\Codeigniter\system\Autoloader\Autoloader.php(296): CodeIgniter\Autoloader\Autoloader->requireFile('C:\\xampp\\htdocs...')
#1 C:\xampp\htdocs\Codeigniter\system\Autoloader\Autoloader.php(258): CodeIgniter\Autoloader\Autoloader->loadInNamespace('App\\Controllers...')
#2 [internal function]: CodeIgniter\Autoloader\Autoloader->loadClass('App\\Controllers...')
#3 [internal function]: spl_autoload_call('App\\Controllers...')
#4 C:\xampp\htdocs\Codeigniter\system\CodeIgniter.php(869): class_exists('\\App\\Controller...', true)
#5 C:\xampp\htdocs\Codeigniter\system\CodeIgniter.php(394): CodeIgniter\CodeIgniter->startController()
#6 C:\xampp\htdocs\Codeigniter\system\CodeIgniter.php(312): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#7 C:\xampp\htdocs\Codeigniter\index.php(46): CodeIgniter\CodeIgniter->run()
#8 {main}
